addons/wowl/static/src/utils/dates.js

```diff
@@ -31,16 +31,19 @@ export function formatDateTime(value, options = {}) {
 
 // -----------------------------------------------------------------------------
 
-const stripAlphaDupesRegex = /([a-zA-Z])(?<=\1[^\1])/g;
-
 /**
  * @param {string} str
  * @returns {string}
  */
 function stripAlphaDupes(str) {
   // Removes any duplicated alphabetic characters in a given string.
   // Example: "aa-bb-CCcc-ddD xxxx-Yy-ZZ" -> "a-b-Cc-dD x-Yy-Z"
-  return str.replace(stripAlphaDupesRegex, "");
```

Comment:
euh, this is incredibly ugly. why can't we use replace? https://caniuse.com/mdn-javascript_builtins_string_replace ??

Reply:
it is the regex stripAlphaDupesRegex = /([a-zA-Z])(?<=\1[^\1])/g; that is not supported by safarri. 
\1 => Parsing error: Octal literal in strict mode